Scripts/Biochemistry/Add_New_Reactions.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
+import os, sys, re, copy
+from csv import DictReader
+from collections import OrderedDict
+temp=list();
+header=1;
+
+Biochem="KEGG"
+Biochem_Root="../../Biochemistry/Aliases/Provenance/Primary_Databases/";
+
+sys.path.append('../../Libs/Python')
+from BiochemPy import Reactions, Compounds, InChIs
+
+CompoundsHelper = Compounds()
+Compounds_Dict = CompoundsHelper.loadCompounds()
+ReactionsHelper = Reactions()
+Reactions_Dict = ReactionsHelper.loadReactions()
+Reactions_Codes = ReactionsHelper.generateCodes(Reactions_Dict)
+
+Default_Rxn = {"id":"cpd00001","name":"null","abbreviation":"null","aliases":"null",
+               "code":"null","stoichiometry":"null","equation":"null","definition":"null",
+               "reversibility":"=","direction":"=","deltag":"10000000","deltagerr":"10000000",
+               "status":"NB","is_obsolete":0,"is_transport":0,
+               "abstract_reaction":"null","pathways":"null","ec_numbers":"null",
+               "compound_ids":"null","linked_reaction":"null","notes":"null","source":""}
+
+Compounds_Alias_Dict=CompoundsHelper.loadMSAliases()
+Source_Alias_Dict = dict()
+for msid in Compounds_Alias_Dict.keys():
```

Comment:
Iterating though keys is the default behavior so you can just do `for msid in Compounds_Alias_Dict:` (I see it several places but I'm not going to spam you by pointing it out every time)